op-program/host/config/config.go

```diff
@@ -52,6 +52,8 @@ var (
 type Config struct {
 	L2ChainID eth.ChainID
 	Rollups   []*rollup.Config
+	// L1ChainConfigs are the op-geth chain config for the L1 execution engine
+	L1ChainConfigs []*params.ChainConfig
```

Comment:
Same thing - I think this should be a single value rather than a map. 

```suggestion
	L1ChainConfig *params.ChainConfig
```

Reply:
This one I am less sure of -- I think the op-program host generally supports multiple chains and they need not have the same L1 (they are not necessarily interoperating). For example, it can be used for op-sepolia and op-mainnet.

Reply:
This does highlight, however, that we may end up having a lot of redundancy here if we store an l1 chain config per l2 chain, since we generally have multiple l2 chains and only a handful (two?) of l1s.

Reply:
You're correct in that the op-program needs to support sepolia and mainnet L1. The rollup config contains the l1_chain_id and this is accessible to the client program. To avoid redundant L1ChainConfigs, we could use a mapping of configs keyed by their l1_chain_id.

Reply:
Nvmind. mbaxter is right that a single config is appropriate here. While a program host could support multiple chains, this only happens in interop mode. And for interop, we expect all chains to use the same L1. So this could be simplified to return a single L1ChainConfig.

Reply:
Thanks for the clarification. I went ahead and used a single config.